packages/types/src/items.d.ts

```diff
@@ -0,0 +1,18 @@
+export type ItemStatus = "available" | "reserved" | "in-use" | "unavailable"; // 예약가능, 예약됨, 대여중, 사용불가
+
+export type ItemType = "seat" | "room" | "equipment";
```

Comment:
```suggestion
export const TItemType = ["seat", "room", "equipment"] as const;
export type TItemType = (typeof TItemType)[number];
```
https://velog.io/@hhhminme/%EB%84%A4-Enum-%EB%88%84%EA%B0%80-Typescript%EC%97%90%EC%84%9C-Enum%EC%9D%84-%EC%93%B0%EB%83%90
타입 안정성과 트리쉐이킹을 위해 as const로 사용하는게 어때
보너스로 type으로 선언하면 prefix로 "T"까지 부착...?